src/components/routes/Host/ListingForm/ListingForm.tsx

```diff
@@ -118,18 +120,37 @@ const ListingFormSchema = Yup.object().shape({
 
 const formCrumbs = ['listing_info', 'accommodations', 'pricing_availability', 'checkin_details'];
 
+const defaultFocus: { [name: string]: string } = {
+  listing_info: 'homeType',
+  accommodations: 'sleepingArrangement',
+  pricing_availability: 'maxGuests',
+  checkin_details: 'checkInTime',
+};
+
+interface AsideHeadersInterface {
+  [name: string]: string;
+}
+
+const AsideHeaders: AsideHeadersInterface = {
+  listing_info: 'Let’s get started! This section will inform guests about where they’ll be staying and what to expect. The more descriptive, the better.',
+  accommodations: 'accommodations header',
```

Comment:
Wondering if we need the word `header` at the end of these?

Reply:
placeholders again -- I'll swap in the actual headers